contracts/mocks/MockLightAccountFactory.sol

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.19;
+
+import {ILightAccountFactory} from "../interfaces/ILightAccountFactory.sol";
+
+contract MockLightAccountFactory is ILightAccountFactory {
+    mapping(address => mapping(uint256 => address)) private _accountAddresses;
+
+    function setAccountAddress(
```

Comment:
You could have fun with this and name it `deployAccount` or `createAccount` to more closely match the actual implementation signature